### PR TITLE
Enable dinov2 ds-ort support

### PR DESCRIPTION
**Pull request by nvijayrania**
Corresponding AzuremlCli PR: https://msdata.visualstudio.com/Vienna/_git/AzureMlCli/pullrequest/1331650

### CODE REVIEW

File: assets/training/finetune_acft_image/src/finetune/finetune.py [hunk 1]
```diff
@@ -693,16 +693,12 @@ def main():
             Setting label_smoothing_factor to 0.0 from {args.label_smoothing_factor}"
```
Comment (rjaincc) [inline assets/training/finetune_acft_image/src/finetune/finetune.py:693]: Can we already bump up the component version, env version in the component and component version in the pipeline components?

**Review by rjaincc — DISMISSED** [review]
Approving with minor comment.

File: assets/training/finetune_acft_image/src/finetune/finetune.py (hunk 1 above)
Comment (nvijayrania) [reply assets/training/finetune_acft_image/src/finetune/finetune.py:693]: Sure. All classification related components are bumped up.

Comment (nvijayrania) [reply assets/training/finetune_acft_image/src/finetune/finetune.py:693]: Updated component versions

File: assets/training/finetune_acft_image/components/model_import/mmd_od_is/spec.yaml [hunk 2]
```diff
@@ -1,14 +1,14 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 type: command
 
-version: 0.0.13
+version: 0.0.14
```
Comment (rjaincc) [inline assets/training/finetune_acft_image/components/model_import/mmd_od_is/spec.yaml:4]: Do we need to bump up version for this? We are not changing anything for MMD model import?

Comment (nvijayrania) [reply assets/training/finetune_acft_image/components/model_import/mmd_od_is/spec.yaml:4]: The envrionment version is getting updated. To use the same env version in model import as well as finetune.

File: assets/training/finetune_acft_image/components/model_import/mmt/spec.yaml [hunk 3]
```diff
@@ -1,14 +1,14 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 type: command
 
-version: 0.0.5
+version: 0.0.6
```
[the remark this answers is not in the document]
Comment (nvijayrania) [reply assets/training/finetune_acft_image/components/model_import/mmt/spec.yaml:4]: The envrionment version is getting updated. To use the same env version in model import as well as finetune.

**Pull request merged**